src/main/webapp/app/admin/user-management/user-management.component.html

```diff
@@ -176,6 +176,9 @@ <h2>
                                 <fa-icon [icon]="faWrench"></fa-icon>
                                 <span class="d-none d-md-inline" jhiTranslate="entity.action.edit">Edit</span>
                             </a>
+                            <a class="btn btn-success btn-sm" (click)="ldapSync(user.login!)">
```

Comment:
This button should only be available when the profile LDAP is active.
Please check this in the ngOnInit method of the corresponding ts file
(there are examples how to load this from profile service)

Reply:
Changes adapted. One quick question, though. When you say available, you mean that the button should only appear to the profiles with LDAP active (*ngIf) or the button should always appear but for those who are not active it should be disabled (like the delete button for the current user)?